app/controllers/payments.js

```diff
@@ -1,19 +1,63 @@
 const config = require('config')
 const stripe = require('stripe')(config.stripe.api_secret)
+const plan = config.stripe.enthusiast_plan_id
```

Comment:
I'll comment more on this in a bit, but I'm not sure what the naming of the plans are yet. I think my idea is to use an abstract name on the back end, like "tier1", which can then be expanded to "tier2", "tier3" etc. We only need to name the tiers on the front-end side.